plugin/adsk/scripts/AETemplateHelpers.py

```diff
@@ -183,7 +183,7 @@ def ProxyShapeFilePathChanged(filePathAttr, newFilePath=None):
             #       in that case.
             primPath = cmds.getAttr(stageName+'.primPath') or ''
             excludedPrimPaths = cmds.getAttr(stageName+'.excludePrimPaths') or ''
-            loadPayloads = cmds.getAttr(stageName+'.loadPayloads') or 0
```

Comment:
Using the new Python load rules helpers to handle the load-payload checkbox in the attribute editor.